torch/csrc/jit/codegen/cuda/scheduler/all_schedulers.h

```diff
@@ -10,6 +10,7 @@ namespace cuda {
 
 enum class TORCH_CUDA_CU_API ScheduleHeuristic {
   None,
+  Null,
```

Comment:
I guess `Null` means the scheduler knows to take no action. `None` means the heuristic is yet to be decided.

Reply:
How about something like path-through instead of null?